server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1158,6 +1159,9 @@ boolean canSuspendTablets() {
       throw new IllegalStateException("Exception setting up FaTE cleanup thread", e);
     }
 
+    ThreadPools.watchCriticalScheduledTask(context.getScheduledExecutor().scheduleWithFixedDelay(
```

Comment:
@dlmarion  did you consider putting this in the GC process?

Reply:
I did not, that's not a bad idea. My only concern is that the GC process is not required to be running, the Manager is. If the GC process died unexpectedly on a cluster and was not noticed for a while, then the references would not be cleaned up. Having said that, the process that uses the references is down. I have no issue moving it to the GC process.

Reply:
If it does move to GC, may be able to add it to the GCs main loop instead of creating another time thread.